parts/k8s/windowsinstallopensshfunc.ps1

```diff
@@ -8,15 +8,23 @@ Install-OpenSSH {
     $adminpath = "c:\ProgramData\ssh"
     $adminfile = "administrators_authorized_keys"
 
-    Write-Host "Installing OpenSSH"
-    $isAvailable = Get-WindowsCapability -Online | ? Name -like 'OpenSSH*'
-
-    if (!$isAvailable) {
-        Write-Error "OpenSSH is not avaliable on this machine"
-        exit 1
+    $sshdService = Get-Service | ? Name -like 'sshd'
+    if ($sshdService.Count -eq 0)
+    {
+        Write-Host "Installing OpenSSH"
+        $isAvailable = Get-WindowsCapability -Online | ? Name -like 'OpenSSH*'
+
+        if (!$isAvailable) {
+            Write-Error "OpenSSH is not avaliable on this machine"
+            exit 1
+        }
+
+        Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0

```

Comment:
@marosset - do you happen to know if this would ever change? It's been the same string in 1803, 1809 and 1903. I don't think we need to change anything in this PR, I'm just wondering how likely this is to break.

Reply:
I don't think it will ever change. I'll reach out to some folks in consine FUN org to find out for sure tho.